src/app/commands/cmd_remove_layer.cpp

```diff
@@ -116,9 +116,14 @@ RemoveLayerCommand::RemoveLayerCommand()
 
 bool RemoveLayerCommand::onEnabled(Context* context)
 {
-  return context->checkFlags(ContextFlags::ActiveDocumentIsWritable |
-                             ContextFlags::HasActiveSprite |
-                             ContextFlags::HasActiveLayer);
+  if (!context->checkFlags(ContextFlags::ActiveDocumentIsWritable |
+                 ContextFlags::HasActiveSprite |
+                 ContextFlags::HasActiveLayer))
+    return false;
+
+  const ContextReader reader(context);
+  const Sprite* sprite(reader.sprite());
+  return sprite && sprite->allLayersCount() > 1;
```

Comment:
Hi @The-Vector, actually here you are counting the total number of layers (including layers in subgroups) which I think it's not the expected behavior, e.g. if you have a group with one child, that is allLayersCount=2 and enables the delete command for the group (which is the only layer at the root level).

You have to count the number of layers in the root (sprite->root()).

Also I'm seeing that the commit has 0quokka0@gmail.com as email but the CLA was signed with other email address, you should sign the CLA with the same name and email that you will use in the commit message.

Reply:
@The-Vector was you able to see the above comments?

Reply:
Yep, I filled out the CLA with the correct email, and I'm just testing my code so there's no other edge cases that I missed.